crates/amaru/src/stages/mod.rs

```diff
@@ -191,13 +192,29 @@ pub fn bootstrap(
     );
 
     let (to_store_header, from_receive_header) = gasket::messaging::tokio::mpsc_channel(50);
-    let (to_validate_header, from_store_header) = gasket::messaging::tokio::mpsc_channel(50);
-    let (to_select_chain, from_validate_header) = gasket::messaging::tokio::mpsc_channel(50);
     let (to_fetch_block, from_select_chain) = gasket::messaging::tokio::mpsc_channel(50);
     let (to_store_block, from_fetch_block) = gasket::messaging::tokio::mpsc_channel(50);
     let (to_ledger, from_store_block) = gasket::messaging::tokio::mpsc_channel(50);
     let (to_block_forward, from_ledger) = gasket::messaging::tokio::mpsc_channel(50);
 
+    // start pure-stage parts, whose lifecycle is managed by a single gasket stage
+    let mut network = TokioBuilder::default();
```

Comment:
This function is already quite long, can we encapsulate the "pure stage" creation logic in function(s)?

Reply:
yes, good point